src/plugins/telemetry_management_section/public/components/telemetry_management_section_wrapper.tsx

```diff
@@ -28,13 +28,15 @@ type Props = any;
 const TelemetryManagementSectionComponent = lazy(() => import('./telemetry_management_section'));
 
 export function telemetryManagementSectionWrapper(
-  telemetryService: TelemetryPluginSetup['telemetryService']
+  telemetryService: TelemetryPluginSetup['telemetryService'],
+  shouldShowSecuritySolutionUsageExample: () => boolean
 ) {
   const TelemetryManagementSectionWrapper = (props: Props) => (
     <Suspense fallback={<EuiLoadingSpinner />}>
       <TelemetryManagementSectionComponent
         showAppliesSettingMessage={true}
         telemetryService={telemetryService}
+        isSecurityExampleEnabled={shouldShowSecuritySolutionUsageExample}
```

Comment:
I am confused here, because `shouldShowSecuritySolutionUsageExample` is a function and `isSecurityExampleEnabled` is a boolean. I feel like shouldShowSecuritySolutionUsageExample should just being a boolean too and you can pass the props `this.showSecuritySolutionExample` into `src/plugins/telemetry_management_section/public/plugin.tsx`